app/views/users/index.html.erb

```diff
@@ -1,11 +1,13 @@
-<%= stylesheet_link_tag "views" %>
+<%= stylesheet_link_tag "users" %>
 <!DOCTYPE html>
 <html>
 <head>
 	<meta charset="utf-8">
 	<title>users</title>
 </head>
 <body>
-  <h1>Here is a list of users</h1>
+  <main id ="container">  
+    <%= render @users %>
```

Comment:
Please make every user a link on `app/views/users/index.html.erb` and when a user clicks on it the page leads to the user's show page.